R/github.R

```diff
@@ -8,7 +8,9 @@ github_auth <- function(token) {
 
 github_response <- function(req) {
   text <- httr::content(req, as = "text")
-  parsed <- jsonlite::fromJSON(text, simplifyVector = FALSE)
+  parsed <-
+    if (identical(text, '')) text else
```

Comment:
This doesn't seem like the right test to me. I think it `httr::has_content()` on `req` would be better. (I presume this was added to handle the delete repo case)